.github/workflows/translations-sync.yml

```diff
@@ -0,0 +1,36 @@
+name: Sync translations
+
+on:
+  workflow_dispatch:
+  push:
+    paths: ['bridge_ui/locales/**']
+    branches: [master]
```

Comment:
This workflow will run either when we manually run it, or when there is any change in the translation files when pushing to master.